### PR TITLE
Fixed an inconsistent treatment of vsys

### PR DESCRIPTION
**Pull request by jinshisai**
- Vsys output on the terminal is now in the absolute value
- Fix a few lines so that Vsys shown on the terminal and in popt is consistent.
- _pvfit_vcut.pdf (_xcut.pdf) --> .pvfit.vcut.png (.xcut.png) (sorted the format to match other files)

### CODE REVIEW

File: pvanalysis/_pvanalysis.py [hunk 1]
```diff
@@ -1152,15 +1143,13 @@ def plot_fitresult(self, vlim: list = [0, 1e10],
         if len(self.popt['ridge'][0]) == 5:
             self.avevsys = (self.popt['edge'][0][4]
                             + self.popt['ridge'][0][4]) / 2.
-            self.vsys += self.avevsys
-            self.popt['edge'][0][4] -= self.avevsys
-            self.popt['ridge'][0][4] -= self.avevsys
+            self.vsys_opt = self.vsys + self.avevsys
         else:
             self.avevsys = 0
```
Comment (yusukeaso-astron) [inline pvanalysis/_pvanalysis.py:1148]: self.vsys_opt is not defined in the case of "else". So, self.avevsys = 0 --> self.vsys_opt = self.vsys?

File: example.py [hunk 2]
```diff
@@ -17,11 +17,11 @@
 vlim_plot = [6. / 20., 6.]  # km/s
 use_velocity = True  # cuts along the velocity direction
 use_position = True  # cuts along the positional direction
-include_vsys = False  # vsys offset. False means vsys=0.
+include_vsys = True  # vsys offset. False means vsys=0.
```
Comment (yusukeaso-astron) [inline example.py:20]: Do we need to change example.py?

File: pvanalysis/_pvanalysis.py [hunk 3]
```diff
@@ -1214,7 +1203,9 @@ def plot_model(self, ax=None, loglog: bool = False,
             print('Please input ax.')
             return -1
         if model is None: model = self.model
-        if popt  == []: popt = self.popt[method][0]
+        if popt  == []:
+            popt = self.popt[method][0].copy()
+            popt[4] -= self.avevsys
```
Comment (yusukeaso-astron) [inline pvanalysis/_pvanalysis.py:1208]: If self.vsys_opt is defined in any case, popt[4] -= self.avevsys --> popt[4] = self.vsys_opt?

File: example.py [hunk 4]
```diff
@@ -21,7 +21,7 @@
 include_dp = True  # False means a single power
 include_pin = False  # False means pin=0.5 (Keplerian).
 show_pv = True  # figures will be made regardless of this option.
-show_corner = True  # figures will be made regardless of this option.
+show_corner = False  # figures will be made regardless of this option.
```
Comment (yusukeaso-astron) [inline example.py:24]: Here is another change.

Comment (yusukeaso-astron) [inline example.py:24]: show_corner = True

**Review by yusukeaso-astron — CHANGES_REQUESTED** [review]

File: example.py (hunk 4 above)
Comment (jinshisai) [reply example.py]: Solved now

**Pull request merged**